common/io/SelectServerTest.cpp

```diff
@@ -582,7 +582,7 @@ void SelectServerTest::testTimeout() {
       10,
       ola::NewSingleCallback(this, &SelectServerTest::ReentrantTimeout, m_ss));
   m_ss->RegisterSingleTimeout(
-      20,
+      40,
```

Comment:
It only logs one timeout in m_timeout_counter not 2. I'm assuming the terminate timeout is triggering before the Reentrant one?